web/src/main/resources/security/securityContext.xml

```diff
@@ -21,11 +21,10 @@
 <beans xmlns="http://www.springframework.org/schema/beans"
     xmlns:security="http://www.springframework.org/schema/security"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xmlns:context="http://www.springframework.org/schema/context" xmlns:p="http://www.springframework.org/schema/p"
-       xmlns:util="http://www.springframework.org/schema/util"
+    xmlns:context="http://www.springframework.org/schema/context"
```

Comment:
Could you see if it is possible to revert back to earlier code-format? It will be easier to see what relevant changes you have made.